### PR TITLE
[TextFields] Getting rid of paddedLabel Class since it's not needed anymore.

### PR DESCRIPTION
**Pull request by mohammadcazig**

### CODE REVIEW

File: components/TextFields/src/MDCTextInputControllerOutlined.m [hunk 1]
```diff
@@ -171,9 +171,9 @@ - (UIBezierPath *)roundedPathFromRect:(CGRect)f
 
   // Draw the path
   [path moveToPoint:CGPointMake(radius + xOffset, yOffset)];
-  [path addLineToPoint:CGPointMake(offset + xOffset - radius, yOffset)];
```
Comment (willlarche) [inline components/TextFields/src/MDCTextInputControllerOutlined.m:174]: Is this related?

**Comment by willlarche** [comment]
I'd update the PR to not have screenshots since there should be no perceivable difference.

File: components/TextFields/src/MDCTextInputControllerOutlined.m (hunk 1 above)
Comment (mohammadcazig) [reply components/TextFields/src/MDCTextInputControllerOutlined.m:174]: Yes the padding needed to be accounted for, and instead of that I was using radius the wrong way.

**Review by willlarche — APPROVED** [review]
